src/Sdk/Sdk.test.js

```diff
@@ -0,0 +1,209 @@
+const { version: packageVersion, repository } = require('../../package.json');
+
+import { Method } from '../Api';
+
+import Sdk from './Sdk';
+
+const BASE_URL = 'https://api.prezly.com';
+const DEFAULT_USER_AGENT = `prezly-javascript-sdk/${packageVersion} (+${repository.url})`;
+const ACCESS_TOKEN = 'example-access-token';
+const DEFAULT_REQUEST_PROPS = {
+    body: undefined,
+    headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json;charset=utf-8',
+        'User-Agent': DEFAULT_USER_AGENT,
+        'x-access-token': ACCESS_TOKEN,
+    },
+};
+
+function successJsonResponse(body, status = 200) {
```

Comment:
This is copied from `Api.test.js`. I wasn't sure if it was worth it to extract this into "test utils"  file.